website/settings/defaults.py

```diff
@@ -174,6 +176,15 @@ def parent_dir(path):
 ## Default RabbitMQ broker
 BROKER_URL = 'amqp://'
 
+CELERY_CREATE_MISSING_QUEUES = False
+CELERY_DEFAULT_QUEUE = 'osf'
+CELERY_QUEUES = (
```

Comment:
When working locally running both the waterbutler celery worker and the OSF celery worker, the two will "steal" tasks from each other and fail because they have entirely different code bases.
Also at some point in time we want to have a single rabbitmq cluster which will make this change even more necessary.